vowpalwabbit/example_predict.cc

```diff
@@ -1,13 +1,12 @@
-/*
```

Comment:
Copyright no longer needed?

Reply:
well... I can't push it to the other codebase

Reply:
well... I can't push it to the other codebase

---
In reply to: [188075530](https://github.com/JohnLangford/vowpal_wabbit/pull/1481#discussion_r188075530) [](ancestors = 188075530)